crates/ibc/src/core/mod.rs

```diff
@@ -33,12 +33,17 @@ pub mod events;
 pub mod timestamp;
 
 mod context;
+mod entrypoint;
 mod handler;
```

Comment:
`handler` and `entrypoint` are basically the same thing; I don't think we need both

Reply:
[4d1ca0a](https://github.com/cosmos/ibc-rs/pull/661/commits/4d1ca0a825b2f37be264c55e2cc9caf5bead0b20)